vnext/Shared/OInstance.cpp

```diff
@@ -579,6 +580,10 @@ void InstanceImpl::loadBundleInternal(std::string &&jsBundleRelativePath, bool s
       m_innerInstance->loadScriptFromString(std::move(bundleString), jsBundleRelativePath, synchronously);
 #endif
 
+#if defined(_CHAKRACORE_H_)
+    } catch (const facebook::react::ChakraJSException &e) {
+      m_devSettings->errorCallback(std::string{e.what()} + "\r\n" + e.getStack());
```

Comment:
can it be `m_devSettings->errorCallback(e.what() + "\r\n" + e.getStack());`

Reply:
No, because we can't concatenate `const char*` types. By converting the first component to std::string, I force the compiler to deduce the other two strings to std::string too.